### DISCUSSION TITLE
How to render revealjs and embed results into quarto website in one go with github actions?

### DISCUSSION THREAD

**Discussion by elsherbini** [quarto-dev/quarto-cli#6939] [Q&A] [answered]
### Description

I'm making a website for a workshop. I'd like to make slides and also link to those slides in the website.

I did this before with RMarkdown by manually knitting slides, and copying the html into a static folder: (see https://github.com/elsherbini/AMNH_R_Workshop_2023). (I'm trying to at least recreate this workflow, and am making some progress with the `include:` block in the `_quarto.yaml`)

Is there a way with profiles, and maybe prerender commands, to have quarto first render presentations, then render the website? I'd really like to do this with github actions so my teammates can quickly iterate on the content for the workshop.

I'd like to have a folder with all the `.qmd` files of slides, have the prerender step render those, and then render the website. 

This is similar to the question here: https://github.com/quarto-dev/quarto-cli/discussions/1751 but adds the github actions angle.

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/output-formats/html-multi-format.html#specifying-formats-to-link> which is to me the simplest way.

Other related discussions:

- https://github.com/quarto-dev/quarto-cli/discussions/6913 (and example at https://github.com/posit-conf-2023/quarto-r)
- https://github.com/quarto-dev/quarto-cli/discussions/1751
- https://github.com/quarto-dev/quarto-cli/discussions/6236
- https://github.com/quarto-dev/quarto-cli/discussions/5849
- https://github.com/quarto-dev/quarto-cli/discussions/3215
